eval.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import scipy.stats as stats
+import torch
+from matplotlib import pyplot as plt
+from tqdm import tqdm
+
+
+def evaluate_model(generator, experiment, test_set, batch_size, batch_num, gan_noise_size, device, scaler):
+    predictions = []
+    for _ in tqdm(range(batch_num), desc='evaluation', position=0, leave=True):
+        x_noise = torch.randn((batch_size, gan_noise_size)).to(device)
+        predictions.append(generator(x_noise).cpu().detach().numpy())
+
+    predictions_np = np.concatenate(predictions)
+
+    inverse_generated = scaler.inverse_transform(predictions_np)
+    test_inverse = scaler.inverse_transform(test_set)
+
+    inverse_generated = inverse_generated[:, [0, 1, 2, 3, 4, 6]]
```

Comment:
magic number 6 is not OK. Please at least make it a named constant.

Reply:
done